### ISSUE TITLE
Custom data, gradient explosion, accuracy is 0

### ISSUE THREAD

**Issue by pbamotra** [huggingface/pytorch-transformers#619]
Hi,

I have 16000+ labels to predict using sequence classifier. I tried running the code with BertAdam (no gradient clipping) and low LR of 1e-5. But my loss doesn't not improve and the accuracy stays at zero. Gradient clipping doesn't help either. I've check my inputs and they are correct. Any help is welcome.

model: base uncased
